apps/docs/src/pages/components/card.mdx

```diff
@@ -22,4 +22,160 @@ export const getStaticProps = async ({ params }) => {
 
 ### Usage
 
+```jsx live
+<HvCard
+  className="w-380px"
+  bgcolor="atmo1"
+  statusColor="positive"
+  icon={<Leaf iconSize="S" color="positive" />}
+>
+  <HvCardHeader
+    title={<HvTypography variant="title4">Schedule New Job</HvTypography>}
+    icon={<Job size="XS" />}
+    subheader={
+      <HvTypography variant="caption2">
+        Run data backups or maintenance tasks on your schedule
+      </HvTypography>
+    }
+  />
+  <HvCardMedia
+    component="img"
+    alt="Schedule New Job"
+    height={120}
+    image="https://github.com/user-attachments/assets/3d17a64e-b5f3-4ea8-a66d-bc02a31b92e0"
+  />
+  <HvCardContent>
+    <HvTypography>
+      Set up a job to run automatically at your preferred time and frequency.
+    </HvTypography>
+  </HvCardContent>
+  <HvActionBar>
+    <div className="flex justify-between w-full">
+      <HvButton variant="secondaryGhost">Configure Job</HvButton>
+      <HvButton variant="secondarySubtle">Set Schedule</HvButton>
+    </div>
+  </HvActionBar>
```

Comment:
Can we instead use a `flex-1` div to separate the buttons? `HvActionBar` already is a flex container, and this way it's easier add/remove buttons & keep the expected behaviour. `justify-between` only works for 2 buttons
```suggestion
  <HvActionBar>
    <HvButton variant="secondaryGhost">Configure Job</HvButton>
    <div aria-hidden className="flex-1" />
    <HvButton variant="secondarySubtle">Set Schedule</HvButton>
  </HvActionBar>
```